pkg/dao/rpms_test.go

```diff
@@ -737,6 +737,19 @@ func (s *RpmSuite) TestInsertForRepositoryWithExistingChecksums() {
 	assert.Equal(t, int64(len(p[1:groupCount+1])), rpm_count)
 }
 
+func (s *RpmSuite) TestInsertForRepositoryWithLotsOfRpms() {
+	t := s.Suite.T()
+	tx := s.tx
+	defer func() { DbInClauseLimit = 60000 }()
+	DbInClauseLimit = 100
```

Comment:
i originally tested this with 70,000 packages, but took ~8 seconds to insert, so i figured just testing that the pagination was correct made more sense here